src/loose/state.js

```diff
@@ -158,4 +158,10 @@ export class LooseParser {
       plugin(this, pluginConfigs[name])
     }
   }
+
+  parse() {
+    let node = this.options.program || this.startNode()
+    this.next()
```

Comment:
I noticed some other differences between loose and strict. The default `Parser` constructor accepts `options, input` but the `LooseParser` accepts `input, options`.